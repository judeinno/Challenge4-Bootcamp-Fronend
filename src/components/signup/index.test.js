import React from 'react';
import { shallow } from 'enzyme';
import SignUp from './index';

it('testing SignUp', () => {
  const SignUpComponent = shallow(
    <SignUp />,
  );
  expect(SignUpComponent).toMatchSnapshot();
});
