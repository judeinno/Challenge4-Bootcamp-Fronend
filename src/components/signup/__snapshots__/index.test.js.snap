// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing SignUp 1`] = `
<div
  className="container signup"
>
  <form
    className="signup-form"
  >
    <legend>
      Register
    </legend>
    <hr />
    <div
      className="form-group"
    >
      <p>
        Username:
      </p>
      <input
        className="form-control"
        id="user-name"
        name="username"
        placeholder="User Name"
        type="test"
      />
    </div>
    <div
      className="form-group"
    >
      <p>
        Email:
      </p>
      <input
        aria-describedby="emailHelp"
        className="form-control"
        id="exampleInputEmail1"
        name="email"
        placeholder="Enter email"
        type="email"
      />
    </div>
    <div
      className="form-group"
    >
      <p>
        Password:
      </p>
      <input
        className="form-control"
        id="exampleInputPassword1"
        name="password"
        placeholder="Password"
        type="password"
      />
    </div>
    <button
      className="btn btn-primary"
      id="sign-up-bt"
      type="submit"
    >
      Submit
    </button>
  </form>
</div>
`;
